add decision making paragraph (#2501)
fastify/fastify: PROJECT_CHARTER.md

### Section 4.2: Decision-making, Voting, and/or Elections

**Decision-making**

> The Fastify's features can be discussed in the GitHub's issues and/or projects.

Reviewer: ```suggestion
Fastify's features can be discussed in GitHub issues and/or projects.
```

> The consensus on a discussion is reached when:

Reviewer: ```suggestion
Consensus on a discussion is reached when:
```

> - the majority of the Collaborator agree on the decision

Reviewer: ```suggestion
- the majority of Collaborators agree on the decision
```

> - there is not any blocking concern by the Lead Maintainers

Reviewer: ```suggestion
- there is not any blocking concern from the Lead Maintainers
```

> 

Reviewer 2: I think we can replace the above two points with there is no objection by any of the collaborators. A concept like majority is complex to manage.

Reviewer 2: @Eomm ping ^

We can lift this from https://github.com/nodejs/node/blob/master/GOVERNANCE.md#collaborators

Whenever there is not consensus, Lead Maintainers will have final say on the topic.

**Voting, and/or Elections**

> Individuals making significant and valuable contributions to the project may be candidate to join
> the Fastify's org.

Reviewer: ```suggestion
Individuals making significant and valuable contributions to the project may be a candidate to join the Fastify organization.
```

> A Fastify Team Member needs to open a private team discussion on GitHub and list the candidates who
> wants to sponsor with a link to the user's contributions (eg: `[USERNAME](https://github.com/search?q=author:USERNAME+org:fastify)`).

Reviewer: ```suggestion
A Fastify Team Member needs to open a private team discussion on GitHub and list the candidates
they want to sponsor with a link to the user's contributions (eg: `[USERNAME](https://github.com/search?q=author:USERNAME+org:fastify)`).
```

> Otherwise, a Contributor may self-apply if he/she thinks to meet the above criteria reaching out
> to a Lead Maintainer privately with a few links to his/her valuable contributions.
> The Lead Maintainer will reply to the Contributor and will decide if candidate he/she to the FTM.

Reviewer: ```suggestion
Otherwise, a Contributor may self-apply if they believe they meet the above criteria by reaching out
to a Lead Maintainer privately with a few links to their valuable contributions.
The Lead Maintainer will reply to the Contributor and will decide if candidate he/she to the FTM.
```

I don't know what this is supposed to mean:

>  will decide if candidate he/she to the FTM.

What is "FTM"?

Author: At the beginning of the file it is specified:

> Technical leadership for the projects within the [OpenJS Foundation][openjs foundation] is delegated to the projects through their project charters by the [OpenJS Foundation Cross-Project Council](https://openjsf.org/about/governance/) (CPC). In the case of the Fastify project, it is delegated to the [Fastify's Team](README.md#team) (FTM).

The consensus to a new candidate to join the FTM is reached when:

> - at least one Lead Maintainers approve
> - at least two Team Members approve
> - no one contrary

Reviewer: ```suggestion
- at least one of the Lead Maintainers approve
- at least two of the Team Members approve
- no one objects
```

Is it possible for a single objection to overrule the first two items?

Author: > Is it possible for a single objection to overrule the first two items?

Theoretically yes.
Do you agree?

Reviewer: No. As long as the first two items are met then the third should not matter. If I dissent on an item, but everyone else is on board with it, then I am outvoted and should accept that.

@fastify/core
